### DISCUSSION TITLE
Support of DryIoC v.5 by PRISM 8 version

### DISCUSSION THREAD

**Discussion by segor** [PrismLibrary/Prism#3091] [General] [answered]
Hello,

We would like to update the obsolete version  DryIoC v4 to the  latest major version v.5 in our project but it is not compatible with PRISM 8. We cannot use Prism 9 version for now in order to it is not released yet and my organization is still in process of purchasing the commercial license. 
Are there any workarounds ? Is it possible to release a new version Prism.DryIoc v8 with new DryIoC support ?

**Comment by dansiegel** [comment, marked as answer]
No we will not be releasing any patches for older releases unless we have an enterprise support client that requires it. And even then it would only be available for them unless they decided to allow it to be released publicly.

**Reply by segor** [reply]
@dansiegel Thanks for the quick response. When we can expect the release of Prism 9 then? I am asking because  we cannot use pre-release versions according to my organization policy even if we buy the commercial license for PRISM 9.

**Reply by dansiegel** [reply]
While I cannot give you an exact date I will say that Commercial Plus users will have it early March.
